doc/pages/mapping.asciidoc

```diff
@@ -110,3 +112,18 @@ be used:
 
 *<f1>*, *<f2>*, ...*<f12>*::
     Function keys.
+
+NOTE: Although Kakoune allows many key combinations to be mapped, not every
+possible combination can be triggered. For example, due to limitations in
+the way terminals handle control characters, mappings like *<c-s-a>* are
+unlikely to work in Kakoune's terminal UI.
+
+== Unsupported keys
```

Comment:
The place to document breakage is the startup message in main. I dont want to litter the documentation with references to how things used to be. (Just a single sentence, "<backtab> is gone, use <s-tab>").